infra/prod/index.ts

```diff
@@ -580,7 +580,18 @@ const celeryWorker = new awsx.ecs.FargateService("scorer-bkgrnd-worker", {
     containers: {
       worker1: {
         image: dockerGtcPassportScorerImage,
-        command: ["celery", "-A", "scorer", "worker", "-l", "DEBUG", "-c", "8"],
+        command: [
+          "celery",
+          "-A",
+          "scorer",
+          "worker",
+          "-Q",
+          "score_registry_passport",
+          "-l",
+          "DEBUG",
+          "-c",
```

Comment:
Not sure we should use `-l DEBUG` in prod ...
Shall we remove this ???

Reply:
I think we should remove it. It was there in prod before, but I don't think it should be.